libs/frontend/feature-build/src/components/toolbar/ToolbarBreakpointOption.vue

```diff
@@ -92,6 +97,7 @@ const breakpointHint = computed(() => {
     @mixin flex-center;
     width: 40px;
     height: 40px;
+    position: relative;
```

Comment:
@abemscac I'm not exactly sure whether it's due to updating `@floating-ui`, or something that happened when I migrated libs, but somehow all the tooltip positions were broken. I had to make sure all the parents had `position: relative;`, it makes sense but I'm not sure why it worked previously.